build.gradle

```diff
@@ -1,15 +1,15 @@
 buildscript {
     ext {
         buildConfig = [
-                'compileSdk': 27,
+                'compileSdk': 28,
                 'minSdk': 19,
                 'buildTools': '28.0.3'
         ]
 
         versions = [
                 'kotlin': '1.2.71',
-                'supportLibrary': '27.1.1',
-                'preferencesFix': '27.1.1.1',
+                'supportLibrary': '28.0.0',
```

Comment:
Is this still necessary, all support library references should be removed. We should mix support library with androidX

Reply:
I tried just to do the minimum necessary changes to keep the changes small. There may be some additional refactorings necessary after the migration.